MapboxNavigation/RouteMapViewController.swift

```diff
@@ -392,16 +389,36 @@ class RouteMapViewController: UIViewController {
         return navigationMapView(mapView, viewFor: annotation)
     }
 
+    func subInstructionsBannerView(for routeProgress: RouteProgress) {
+        let visualInstructionIndex = routeProgress.currentLegProgress.currentStepProgress.visualInstructionIndex
+        
+        if let visualInstructions = routeProgress.currentLegProgress.currentStep.instructionsDisplayedAlongStep,
+               visualInstructionIndex < visualInstructions.count,
+           let tertiaryInstruction = visualInstructions[visualInstructionIndex].tertiaryInstruction {
+            
+            if tertiaryInstruction.components.contains(where: { $0 is LaneIndicationComponent }) {
```

Comment:
I wonder if we could clean this up further, this check is happening also again on `LanesView.update(for:)`:

https://github.com/mapbox/mapbox-navigation-ios/blob/67f0ab515f76e5855998acf40fdc66c0d91c83f3/MapboxNavigation/LanesView.swift#L69-L85